doc/examples/transform/plot_pyramid.py

```diff
@@ -19,7 +19,8 @@
 
 image = data.astronaut()
 rows, cols, dim = image.shape
-pyramid = tuple(pyramid_gaussian(image, downscale=2, channel_axis=-1))
+pyramid_generator = pyramid_gaussian(image, downscale=2, channel_axis=-1)
```

Comment:
Generators are a commonly used, built-in part of the language, so I wouldn't separate out these steps. I'm fine with making it clearer in the docs.

Reply:
I am 👍🏻 on @stefanv's comment. Otherwise, looks good-to-go.